scripts/setup_compute_stack.sh

```diff
@@ -259,11 +259,14 @@ configure_nevermined_compute() {
     $K create namespace $COMPUTE_NAMESPACE
   fi
 
+  #mount local directory to minikube vm
+#  minikube mount  ${KEEPER_ARTIFACTS_FOLDER}:/tmp/hostpath-provisioner/nvm-disc/argo-artifacts &
+
   #echo -e "${COLOR_B}Creating configmap with artifacts from folder ${KEEPER_ARTIFACTS_FOLDER} ...${COLOR_RESET}"
   #$K create -n $COMPUTE_NAMESPACE configmap artifacts --from-file=${KEEPER_ARTIFACTS_FOLDER}  
 
   # Install argo artifacts
-  helm install -n $COMPUTE_NAMESPACE argo-artifacts stable/minio --set fullnameOverride=argo-artifacts --set resources.requests.memory=1Gi
+  helm install -n $COMPUTE_NAMESPACE argo-artifacts bitnami/minio --version 11.10.13   --set fullnameOverride=argo-artifacts --set resources.requests.memory=1Gi
```

Comment:
minio version should be defined in an environment variable like `MINIKUBE_VERSION` and not hardcoded here